.eslintrc

```diff
@@ -1,33 +1,24 @@
 {
-  "rules": {
-    "indent": [2, 2, {
-      "VariableDeclarator": {
-        "var": 2,
-        "let": 2,
-        "const": 3
-      }
-    }],
-    "no-empty": [2],
-    "no-console": [0],
-    "comma-dangle": [2],
-    "no-unused-vars": [0],
-    "linebreak-style": [2, "unix"],
-    "semi": [2, "always"]
-  },
+  "parser": "babel-eslint",
+  "extends": "./node_modules/mofo-style/.eslintrc.yaml",
   "env": {
+    "node": true,
     "commonjs": true,
     "es6": true,
     "browser": true
   },
-  "extends": "eslint:recommended",
   "parserOptions": {
+    "ecmaVersion": 6,
     "ecmaFeatures": {
-      "es6": true,
-      "jsx": true,
-      "experimentalObjectRestSpread": true
+      "jsx": true
     }
   },
+  "rules": {
+    "no-console": 1,
+    "react/jsx-uses-react": 1,
```

Comment:
oh I see. Yeah in that case this is a good idea, and perhaps worth adding to a ".eslint-with-react" over on the mofostyle project